assets/i18n/en.json

```diff
@@ -491,7 +491,7 @@
     "I18N_SIGNUP_EMAIL_PREFERENCES_EXPLAIN": "You can change this setting at any time from your Preferences page.",
     "I18N_SIGNUP_ERROR_MUST_AGREE_TO_TERMS": "In order to edit explorations on this site, you will need to agree to the site terms.",
     "I18N_SIGNUP_ERROR_NO_USERNAME": "Please enter a username.",
-    "I18N_SIGNUP_ERROR_USERNAME_MORE_50_CHARS": "A username can have at most 50 characters.",
+    "I18N_SIGNUP_ERROR_USERNAME_MORE_30_CHARS": "A username can have at most 30 characters.",
```

Comment:
Please interpolate, do not hardcode the 30. Otherwise if we change it in the future then we have to go through all this again.

Reply:
Done.